server/app/services/reporting/ApplicationSubmissionsStat.java

```diff
@@ -30,8 +32,11 @@ public static ApplicationSubmissionsStat create(
         submissionDurationSeconds99p);
   }
 
-  /** The name of the program the applications were submitted for. */
-  public abstract String programName();
+  /** The public-facing name of the program. */
+  public abstract String publicName();
+
+  /** The URL name of the program the applications were submitted for. */
+  public abstract String programURLName();
```

Comment:
suggestion (blocking): please follow style guide for naming: https://google.github.io/styleguide/javaguide.html#s5.3-camel-case

```suggestion
  public abstract String programUrlName();
```